scripts/build-asm.py

```diff
@@ -28,6 +28,10 @@
     ('linux', 'x86', 'elf', ['-fPIC', '-DOPENSSL_IA32_SSE2'], 'S'),
     ('linux', 'x86_64', 'elf', [], 'S'),
     ('mac', 'x86_64', 'macosx', [], 'S'),
+    # ('windows', 'aarch64', 'coff', [], 'S'),
+    # ('windows', 'arm', 'coff', [], 'S'),
```

Comment:
Is there a 32-bit ARM version of Windows?

Reply:
Yes, though I've only added support for the modern ARM NT to clang, llvm, and swift, and not Windows CE.  Current windows only supports x86, x64, armv7, aarch64.